pyproject.toml

```diff
@@ -31,24 +31,24 @@ packages = [ {include = "pypfopt"} ]
 "Personal website" = "https://reasonabledeviations.com"
 
 [tool.poetry.dependencies]
-python = "^3.5"
+python = "^3.6"
```

Comment:
This is the only compromise I had to make. Notice, however, that python 3.5 is also not listed above:
```
"Programming Language :: Python :: 3.8",
"Programming Language :: Python :: 3.7",
"Programming Language :: Python :: 3.6",
"Programming Language :: Python :: 3 :: Only",
```
If you think it is crucial to keep python 3.5 working, I am sure one could find a workaround (e.g. allowing for an older version of matplotlib).